src/clr-addons/quick-list/add-option/add-option.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="clr-row">
+  <div class="clr-col-10 pl-0">
+    <clr-select-container>
+      <select clrSelect name="options" [ngModel]="selectedValue" (ngModelChange)="selectedValue = $event">
+        <option *ngFor="let val of getValues()"
+                [selected]="val.id === selectedValue" [disabled]="!val.value" [ngValue]="val.id">
+          {{val.label}}
+        </option>
+      </select>
+    </clr-select-container>
+  </div>
+  <div class="clr-col-2 pl-0">
+    <button class="btn btn-icon btn-link btn-thrash">
```

Comment:
trash button doesn't work at all...

Reply:
click handler needs to be set on the button, not the icon, otherwise this doesn't work in firefox